src/workerd/io/actor-cache-test.c++

```diff
@@ -1188,11 +1185,14 @@ KJ_TEST("ActorCache flush retry") {
     // last time, because it hasn't been further overwritten, and that delete from last time
     // wasn't actually committed.
     mockTxn->expectCall("delete", ws)
-        .withParams(CAPNP(keys = ["grault", "corge"]))
+        .withParams(CAPNP(keys = ["quux"]))
+        .thenReturn(CAPNP());  // count ignored because we got it on the first try!
+    mockTxn->expectCall("delete", ws)
+        .withParams(CAPNP(keys = ["corge", "grault"]))
         .thenReturn(CAPNP(numDeleted = 2));
     mockTxn->expectCall("delete", ws)
-        .withParams(CAPNP(keys = ["quux", "baz"]))
-        .thenReturn(CAPNP(numDeleted = 1234));  // count ignored
+        .withParams(CAPNP(keys = ["baz"]))
+        .thenReturn(CAPNP(numDeleted = 1234));
```

Comment:
I'm confused by the number regardless of the comment, what is the significance of numDeleted = 1234? I see baz is counted immediately so this would do a regular delete, but in that case why not exclude numDeleted entirely? Does including numDeleted with a clearly unexpected value imply the value is not checked better than excluding it completely?

Reply:
> Does including numDeleted with a clearly unexpected value imply the value is not checked better than excluding it completely?

¯\_(ツ)_/¯ that's a fair question. In cases like this where we've already verified that the `delete()` call returned true, it's serving no purpose. In a case where perhaps the delete call didn't return immediately (e.g. because multiple keys were provided) there's a bit more of an argument for returning an unexpected number rather than not returning a count, since not returning a count is equivalent to returning 0 and we might fail to notice if a count of 0 is being used incorrectly.

But in either case, I think your understanding here is correct.